compose-builder/add-security.yml

```diff
@@ -97,49 +106,24 @@ services:
     image: ${CORE_EDGEX_REPOSITORY}/docker-security-secretstore-setup-go${ARCH}:${CORE_EDGEX_VERSION}${DEV}
     container_name: edgex-vault-worker
     hostname: edgex-vault-worker
```

Comment:
IMO, we should change these to `edgex-secretstore-setup` and the service name above to `secretstore-setup` so names are consistent.

Reply:
changed